src/langserver.ts

```diff
@@ -0,0 +1,50 @@
+import { LanguageClient, LanguageClientOptions, ServerOptions, TransportKind } from "vscode-languageclient";
+import { clearTempFolder, getWorkspaceConfig } from "./utils";
+import * as net from "net";
+import { ExtensionContext, window, StatusBarAlignment, workspace } from "vscode";
+
+export let client: LanguageClient;
+export function activateLSP(context: ExtensionContext) {
+	let prepareStatus = window.createStatusBarItem(StatusBarAlignment.Left);
+	console.log("Commencing V language server...");
+	if (!getWorkspaceConfig().get("vls.enable")) return;
+	// Path to VLS executable.
+	const serverPath: string = getWorkspaceConfig().get("vls.path");
```

Comment:
no type needed, typescript will infer the type.

Reply:
Typescript will complain if i reference it inside `serverOptionsStd`

Reply:
k